app/application/airflow/dags/send_msg_1_dag.py

```diff
@@ -0,0 +1,26 @@
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+from logic.send_random_message import send_random_image_and_text
+
+with DAG(
+    dag_id='send_quote_first_approach',
+    schedule=None,
+    tags=['cheer_up_your_colleagues'],
+    description='lorem ipsum dolor sit amet, consectetur adipiscing',
+    catchup=False,
+) as first_approach:
+    start = EmptyOperator(
```

Comment:
No need to break to separate lines such a short string

Reply:
Fixed in [next commit](https://github.com/imPDA/CheerUpYourColleagues/pull/3/commits/84820a51e65bc1fe5a4463808d627342226c0054)